### PR TITLE
show username from moodle

### PR DESCRIPTION
**Pull request by khalil2535**
- [x] create TextView in main_page.xml
- [x] parse the request to get the username and show it 
- [x] remove allways json from form request body

### CODE REVIEW

File: app/src/main/java/ps/edu/ucas/ucashub/services/web/MoodleServices.java [hunk 1]
```diff
@@ -9,33 +9,46 @@
...
     private static final String PASSWORD_ID = "password";
-    private static final String JSON = "json";
+    //Saved responses
+    private static String HOME_PAGE_RESPONSE;
 
     private MoodleServices() {
     }
 
+    private static String getUserID() {
+        Log.d(TAG, "getUserID");
```
Comment (WaleedMortaja) [inline app/src/main/java/ps/edu/ucas/ucashub/services/web/MoodleServices.java]: Dublicated log. It is repeated at line 38

File: app/src/main/java/ps/edu/ucas/ucashub/services/web/MoodleServices.java [hunk 2]
```diff
@@ -85,4 +98,16 @@ public static boolean Login(String username, String password)
         }
         return getAuthorization(username, password) == Auth.ACCEPTED;
     }
+
+    public static String getUserName() {
+        Log.d(TAG, "getUserName");
```
Comment (WaleedMortaja) [inline app/src/main/java/ps/edu/ucas/ucashub/services/web/MoodleServices.java]: Dublicated log.

File: app/src/main/java/ps/edu/ucas/ucashub/services/web/MoodleServices.java [hunk 3]
```diff
@@ -9,33 +9,46 @@
...
-    private static final String JSON = "json";
+    //Saved responses
+    private static String HOME_PAGE_RESPONSE;
 
     private MoodleServices() {
     }
 
+    private static String getUserID() {
+        Log.d(TAG, "getUserID");
+        String userID = HOME_PAGE_RESPONSE.substring(
```
Comment (WaleedMortaja) [inline app/src/main/java/ps/edu/ucas/ucashub/services/web/MoodleServices.java]: You may check
 if (HOME_PAGE_RESPONSE != null)

**Comment by khalil2535** [comment]
all changes are in #19

**Pull request closed without merging**